src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -743,10 +746,15 @@ export default {
 
 	directives: {
 		Focus,
-		Linkify,
 		ClickOutside,
 	},
 
+	provide() {
+		return {
+			'NcAppSidebar:header:id': computed(() => this.headerId),
```

Comment:
It is a static value

```suggestion
			'NcAppSidebar:header:id': this.headerId,
```